src/find-options/FindOperator.ts

```diff
@@ -135,4 +137,27 @@ export class FindOperator<T> {
 
         return this._getSql
     }
+
+    transformValue(transformer: ValueTransformer | ValueTransformer[]) {
+        if (this._value instanceof FindOperator) {
+            this._value.transformValue(transformer)
+        } else {
+            // do not convert each value of json, since an array is a valid json object
+            this._value =
+                Array.isArray(this._value) &&
+                this["@instanceof"] !== Symbol.for("JsonContains")
```

Comment:
This is an anti-pattern.. the general `FindOperator` shuoldn't know about `JsonContains`..
`JsonContains` can override `transformValue` maybe.. and call `super` if needed

Reply:
@vlad-ostrovski  I agree... I was lazy.. I will fix it.

Reply:
fixed!